docs/production-deployment/cloud/worker-health.mdx

```diff
@@ -185,14 +185,16 @@ Consider the following:
   - Golang: [Workflows](/develop/go/core-application#maxconcurrentworkflowtaskpollers)
 - Doing some combination of the above
 
-** Temporal Cloud bottleneck**
+**Temporal Cloud bottleneck**
 
 If the Schedule-To-Start latency is _high_ and the Sync Match Rate is also _low_, Temporal Cloud could very well be the bottleneck and you should reach out via support channels for us to confirm.
 
-**Caveat**
+:::warning
```

Comment:
For some reason in my head, I was sure this was "caution" but I looked it up in the style guide and it's warning.